diff -ruN lnmp/lnmp.conf lnmp-master/lnmp.conf
--- lnmp/lnmp.conf	2023-12-06 12:20:00.068659842 +0800
+++ lnmp-master/lnmp.conf	2023-12-06 12:21:54.000000000 +0800
@@ -15,9 +15,9 @@
 Enable_Nginx_Openssl='y'
 Enable_Nginx_Lua='n'
 Enable_Ngx_FancyIndex='n'
-Enable_Swap='y'
-Enable_PHP_Exif='n'
-Enable_PHP_Fileinfo='n'
+Enable_Swap='n'
+Enable_PHP_Exif='y'
+Enable_PHP_Fileinfo='y'
 Enable_PHP_Ldap='n'
 Enable_PHP_Bz2='n'
 Enable_PHP_Sodium='n'
